habitat_sim/simulator.py

```diff
@@ -119,45 +122,45 @@ def close(self):
 
         super().close()
 
-    def __enter__(self):
+    def __enter__(self) -> "Simulator":
         return self
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def __exit__(self, exc_type: None, exc_val: None, exc_tb: None) -> None:
```

Comment:
Hmm, this might be a mistake and should be Any, I think. Might want to consult the Python docs to see what the proper arg typing for this is.